README.md

```diff
@@ -18,7 +18,7 @@ Register as "screenshot" (see below)
 plugins: {
   screenshot: {
     module: 'nemo-screenshot',
-    arguments: ['path:report']
+    arguments: [argument]
```

Comment:
To be consistent with other method signatures in the `nemo` family, it would ideally be:
`arguments: [reportPath {String}[, eventArray {Array}]]`.